playwright-e2e/pages/singular/enrollment/pre-screening-page.ts

```diff
@@ -59,10 +59,11 @@ export default class PreScreeningPage {
 
   async checkReCaptcha(): Promise<void> {
     const iframe = this.page.frameLocator('css=iframe[title="reCAPTCHA"]');
+    await iframe.locator('css=span[role="checkbox"]').waitFor({ state: 'visible' });
     await iframe.locator('css=span[role="checkbox"]').dispatchEvent('click');
     await iframe
       .locator('.recaptcha-checkbox-spinner[style*="animation-play-state: running;"]')
-      .waitFor({ state: 'hidden' });
+      .waitFor({ state: 'hidden', timeout: 30 * 1000 });
```

Comment:
This failed once on localhost, network speed was slow.

Reply:
Could the reasoning for using `30s` be added a comment so future devs aren't left wondering why this is the particular magic value to be used?

Reply:
No particular reason. 30 sec is a common value for timeout in test automation, normally based on most network requests duration times. It's not long and not short.

Reply:
Sorry- to clarify, I meant as a comment in the code itself (especially if the default is 10s)

Reply:
My fault. I should have explained as a comment. I'll add it in a follow-up PR. Also, I'll add comments for few other places where 30 sec are also used.

Reply:
No worries- thank you for the details!